main.py

```diff
@@ -153,6 +163,7 @@ def add_policy():
     """
     logging.debug(json.dumps(request.json))
     name = request.json["name"]
+    display_name = request.json["displayname"] or name
```

Comment:
Dict access via this syntax will throw a KeyError if the key is not present. Try this instead `request.json.get("displayname",name)`